File: Week-1-C/Assignments/cash/cash.c
#import <cs50.h>
#import <stdio.h>

int main(void)
{
    // Set variables.
    int change;
    int coins = 0;

    // Repeat question to user untill answer is correct (possitive number).
    do
    {
        change = get_int("Change owed: ");
    }
    while (change < 1);

    // While change is not equal to 0, check change value against value of coins.
    // take away value of coin from change and add 1 to coins if used.
    do
    {
        if (change >= 25)
        {
            change = change - 25;
            coins++;
        }
        else if (change >= 10)
        {
            change = change - 10;
            coins++;
        }
        else if (change >= 5)
        {
            change = change - 5;
            coins++;
        }
        else
        {
            change = change - 1;
            coins++;
        }
    }
    while (change > 0);

    // Print coins used.
    printf("%i\n", coins);
}

File: Week-6-Python/Assignments/sentimental-cash/cash.py
from cs50 import get_float

# Ask for users input until the answer is a positive float.
while True:
    change = get_float("Change: ")
    if change > 0:
        break

# set variable coins to 0.
coins = 0

# while change is not equal to 0, check if each coin against change.
while True:
    change = round(change, ndigits=2)

    if change >= 0.25:
        change -= 0.25
        coins += 1
    elif change >= 0.10:
        change -= 0.10
        coins += 1
    elif change >= 0.05:
        change -= 0.05
        coins += 1
    elif change >= 0.01:
        change -= 0.01
        coins += 1
    elif change == 0:
        print(f"{coins}")
        break
